lib/Interactions/Internal/WebDriverSingleKeyAction.php

```diff
@@ -20,5 +27,9 @@ public function __construct(
     ) {
         parent::__construct($keyboard, $mouse, $location_provider);
         $this->key = $key;
+
+        if (!in_array($key, self::MODIFIER_KEYS, true)) {
```

Comment:
I'd put this assert beggining of the method, right after parent constructor call - this is usual place to put check for input conditions.